package.json

```
@@ -0,0 +1,57 @@
+{
+  "name": "@idea2app/next-shadcn-ts",
+  "version": "0.1.0",
+  "description": "React project scaffold based on TypeScript, Next.js, shadcn/ui & Tailwind CSS.",
+  "private": true,
+  "dependencies": {
+    "class-variance-authority": "^0.7.1",
+    "clsx": "^2.1.1",
+    "lucide-react": "^0.475.0",
+    "next": "15.1.7",
+    "react": "^19.0.0",
+    "react-dom": "^19.0.0",
+    "tailwind-merge": "^3.0.1",
+    "tailwindcss-animate": "^1.0.7",
+    "@formatjs/intl-localematcher": "^0.6.0",
+    "negotiator": "^1.0.0"
+  },
+  "devDependencies": {
+    "@eslint/eslintrc": "^3.3.0",
+    "@tailwindcss/postcss": "^4.0.8",
+    "@types/node": "^20.17.19",
+    "@types/react": "^19.0.10",
+    "@types/react-dom": "^19.0.4",
+    "@types/negotiator": "0.6.3",
+    "eslint": "^9.21.0",
+    "eslint-config-next": "15.1.7",
+    "eslint-config-prettier": "^10.0.1",
+    "eslint-plugin-simple-import-sort": "^12.1.1",
+    "@cspell/eslint-plugin": "^8.17.4",
+    "prettier-plugin-css-order": "^2.1.2",
+    "prettier-plugin-tailwindcss": "^0.6.11",
+    "husky": "^9.1.7",
+    "lint-staged": "^15.4.3",
+    "tailwindcss": "^4.0.8",
+    "typescript": "^5.7.3"
+  },
+  "prettier": {
+    "plugins": [
+      "prettier-plugin-css-order",
+      "prettier-plugin-tailwindcss"
+    ]
+  },
+  "lint-staged": {
+    "*.{html,md,less,json,yml,js,mjs,ts,tsx}": "prettier --write",
+    "*.{css,js,mjs,ts,tsx}": "eslint --fix"
+  },
+  "scripts": {
+    "dev": "next dev --turbopack",
+    "build": "next build",
+    "start": "next start",
+    "lint": "next lint",
+    "lint:all": "eslint --fix .",
```

```suggestion
    "lint": "next lint --fix",
```
Next.js 封装的命令就是处理全部。